packages/shared/src/components/sidebar/ClickableNavItem.tsx

```diff
@@ -32,17 +37,20 @@ export function ClickableNavItem({
 
   if (!isButton && (!item.action || item.path)) {
     return (
-      <Link href={item.path} passHref prefetch={false}>
-        <a
-          {...(item.action && { onClick: item.action })}
-          {...props}
-          target={item?.target}
-          className={navBtnClass}
-          rel="noopener noreferrer"
-        >
-          {children}
-        </a>
-      </Link>
+      <>
+        <Link href={item.path} passHref prefetch={false}>
+          <a
+            {...(item.action && { onClick: item.action })}
+            {...props}
+            target={item?.target}
+            className={navBtnClass}
+            rel="noopener noreferrer"
+          >
+            {children}
+          </a>
+        </Link>
+        {isValidElement(item.badge) && item.badge}
```

Comment:
If it's inside the anchor element then the whole tooltip body will be a link to the `https://app.daily.dev/sources/daily_updates` which is bad because we have other CTA and links inside the changelog tooltip.